openvm/tests/apc_builder_outputs/beqz.txt

```diff
@@ -24,7 +24,7 @@ Symbolic machine using 14 unique main columns:
 
 // Bus 0 (EXECUTION_BRIDGE):
 mult=is_valid * -1, args=[0, reads_aux__1__base__prev_timestamp_0 + reads_aux__1__base__timestamp_lt_aux__lower_decomp__0_0 + 131072 * reads_aux__1__base__timestamp_lt_aux__lower_decomp__1_0]
-mult=is_valid * 1, args=[8 - (4 * a__0_0 * diff_inv_marker__0_0 + 4 * a__1_0 * diff_inv_marker__1_0 + 4 * a__2_0 * diff_inv_marker__2_0 + 4 * a__3_0 * diff_inv_marker__3_0), reads_aux__1__base__prev_timestamp_0 + reads_aux__1__base__timestamp_lt_aux__lower_decomp__0_0 + 131072 * reads_aux__1__base__timestamp_lt_aux__lower_decomp__1_0 + 2]
+mult=is_valid * 1, args=[4, reads_aux__1__base__prev_timestamp_0 + reads_aux__1__base__timestamp_lt_aux__lower_decomp__0_0 + 131072 * reads_aux__1__base__timestamp_lt_aux__lower_decomp__1_0 + 2]
```

Comment:
This seems wrong, we turned the conditional jump into an unconditional one.

Reply:
Could be correct, no?